my-dsl.md

```diff
@@ -1,38 +1,68 @@
 # Language
-_What is the name of the language? Link the name to its webpage 
-(if appropriate)._
+_What is the name of the language? Link the name to its webpage
+ (if appropriate)._
+
+The launchd system. 
 
 
 # Domain
 _Describe the language's domain in five words._
 
+Managing processes/scripts for OSX.
+
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a program in your DSL runs._
 
+On startup, launchd reads property lists files in certain directories. Then
+depending on the property list (`plist`), the process is run at the specified
+intervals or otherwise stopped or managed.
+
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
+launchd is "purely" domain-specifice. A launchd property list has less than
+20 possible properties. I don't think you can set variables in these `plist`s.
+In terms of use, people only use it to manage processes and scripts written
+in other languages.
+
```

Comment:
It's pretty common now to have DSLs that aren't directly written by humans. Instead, humans interact with tools (e.g., graphical front-ends) that generate the programs in the DSL. I'm not sure if there are such tools for launchd. If not, we could certainly say that launchd's infrastructure or environment (i.e., the language _plus_ any tools to support it) is not user-friendly; but I would still call it a DSL.